### DISCUSSION TITLE
Cannot view scenes in admin panel

### DISCUSSION THREAD

**Discussion by flumflam2** [mozilla/hubs#6050] [General Questions & Discussion]
When I try to view my scenes in any tab in the admin panel, the site tells me to log in again and sends me a link. 
When I do that and click on the link, nothing happens and I would have to do it again. 

Since a few days ago, when I try to create a room, it loads a long time, then tells me the room cannot be created. Now I cannot check if the reason is a corrupted or large scene.

Please help!

**Comment by matthewbcool** [comment]
Is this Hubs Cloud? Are there any errors in the console?

**Reply by flumflam2** [reply]
Its not self hosted, so its not Hubs Cloud right? I access the panel through the dashboard from the browser

**Reply by matthewbcool** [reply]
perhaps you are using the subscription service then? If that is the case, please create a [ticket with support](https://support.mozilla.org/en-US/products/hubs) so they can assist you: 

Also welcome to post any errors and scene links here to see if we can determine the cause of the issue you are seeing. Sorry to hear about the trouble.
